go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/containerd/continuity v0.0.0-20190426062206-aaeac12a7ffc // indirect
 	github.com/containernetworking/cni v0.7.1
 	github.com/containernetworking/plugins v0.8.1
-	github.com/containers/buildah v1.10.1
+	github.com/containers/buildah v1.8.4-0.20190821140209-376e52ee0142
```

Comment:
@vrothberg So I grabbed the top of buildah/master, and it appears to have downgraded, version-wise. We might want to look into that?

Reply:
The important part is behind the last dash (i.e., the commit `376e52ee0142`). It's a bug in go (no joke) which is trying to match a commit to a tag.

Commit 376e52ee0142 looks good.

Reply:
If we want to get rid of that, we can do `podman run --privileged -v `pwd`:/src:Z -it --rm golang:1.13-rc-buster` and `go get github.com/containers/buildah@376e52ee0142` inside the container. Go 1.13 will set the correct version/tag.

The annoying thing about that bug is that go mod's min-version algorithm can break here and downgrade a whole bunch of other dependencies.